aidies-project/client/src/components/CampaignBrowser.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import { Redirect, Link } from 'react-router-dom';
+
+import {getCampaigns} from '../store/campaign';
+import { useSelector, useDispatch } from 'react-redux';
+
+const CampaignBrowser = () => {
+    const campaigns = useSelector(state => state.campaign.list );
+    const dispatch = useDispatch();
+    useEffect(() => {
+        dispatch(getCampaigns())
+    }, [dispatch]);
+    
+    return (
+        <main>
+            {campaigns.map((campaign) => {
+                return (
+                    <div className="nav-entry">
+                        <Link key={campaign.id} to={`/campaign/${campaign.id}`}>
+                            <img
+                                className="nav-campaign-image"
+                                src={campaign.image}
+                                alt={campaign.name}
+                                width="300"
```

Comment:
Since you have a `className` for this image, have the `width` defined in the CSS for this class!

Reply:
Cool. This is mainly for testing purposes so I don't see very large images. We'll get rid of it